application/ai/tension_analyzer_llm_contract.py

```
@@ -0,0 +1,100 @@
+class TensionAnalyzerLlmPayload(BaseModel):
+    """张力分析器 LLM 响应根对象。
+
+    仅允许四个字段：diagnosis / tension_level / missing_elements / suggestions。
+    """
+
+    model_config = ConfigDict(extra="forbid")
+
+    diagnosis: str = Field(
+        default="",
+        max_length=2000,
+        description="张力诊断描述（2-3 句话）",
```

Ruff (0.15.10)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)